### PR TITLE
Disable TCP Timestamp

### PR DESCRIPTION
**Pull request by Acenl12**
the TCP timestamp response to approximate the remote host's uptime and aid in further attacks. Additionally, some operating systems can be fingerprinted based on the behavior of their TCP time stamps.

### CODE REVIEW

**Comment by konstruktoid** [comment]
Hi again @Acenl12 
Any updates?

File: defaults/main/ufw.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-ufw_enable: true
+ufw_enable: false
```
Comment (konstruktoid) [inline defaults/main/ufw.yml:2]: This shouldn't be changed.

**Review by konstruktoid — CHANGES_REQUESTED** [review]
Fix `ufw` change.

**Comment by Acenl12** [comment]
Oops was a mistake

**Pull request closed without merging**